pdns_api.sh

```diff
@@ -97,6 +97,38 @@ load_config() {
     . "${CONFIG}"
   fi
 
+  if [[ -n "${CONFIG_D}" ]]; then
```

Comment:
I think this should be `${CONFIG_D-}` to resolve an unbound variable error when `CONFIG_D` is not set.

Reply:
Done :)